src/commands/show.ts

```diff
@@ -111,7 +111,7 @@ async function showProblemInternal(node: IProblem): Promise<void> {
 
         const originFilePath: string = await leetCodeExecutor.showProblem(node, language, outDir);
         const filePath: string = wsl.useWsl() ? await wsl.toWinPath(originFilePath) : originFilePath;
-        await vscode.window.showTextDocument(vscode.Uri.file(filePath), { preview: false });
+        await vscode.window.showTextDocument(vscode.Uri.file(filePath), { preview: false, viewColumn: vscode.ViewColumn.One });
```

Comment:
I want to fix the extension usage to use Column 1 as text editor, column 2 for grouping WebView panels (Preview, Solution, and testcase submission).

Reply:
Splitting them is fine to me. My concern is that if the preview is showed in column 2 is a good design. Since in the future you may want to remove the comments in the generated file. The user may need to preserve the preview page in the window.